src/test/java/uk/gov/hmcts/cmc/claimstore/utils/BeanValidator.java

```diff
@@ -20,8 +23,17 @@ private BeanValidator() {
 
     private static <T> Set<String> getMessages(final Set<ConstraintViolation<T>> response) {
         return response.stream()
-            .map(r -> r.getPropertyPath() + " : " + r.getMessage())
+            .map(r -> prepareMessage(r.getPropertyPath(), r.getMessage()))
```

Comment:
Looks like `prepareMessage()` could be refactored to accept one argument: `response`, and then query it for values, so that the line would look like:
```java
.map(this::convertValidationResponseToMessage)
```